avidemux_ru.ts

```diff
@@ -9287,12 +9291,12 @@ The current editing has been saved and will be reloaded at next start</source>
     <message>
         <location line="+2"/>
         <source>Image Size:	</source>
-        <translation>Размер картинки:</translation>
+        <translation>Размер картинки: </translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Aspect Ratio:	</source>
-        <translation>Соотношение сторон:</translation>
+        <translation>Соотношение сторон:	</translation>
```

Comment:
The purpose of the discrepancy between the English sources in Q_props.cpp and Russian translations ("missing" tabs) was to fix text alignment when copying the properties of the video to clipboard.

Reply:
I'll revert it